src/sql/workbench/services/objectExplorer/browser/connectionTreeAction.ts

```diff
@@ -362,5 +365,6 @@ export class RemoveFilterAction extends Action {
 			await this._tree.refresh(nodeToRefresh);
 			await this._tree.expand(nodeToRefresh);
 		}
+		this._telemetryService.createActionEvent(TelemetryKeys.TelemetryView.ObjectExplorer, TelemetryKeys.TelemetryAction.ObjectExplorerRemoveFilter)
```

Comment:
1. You're never calling "send" so this won't be sent
2. Don't you want to add in object type or something to identify where this filter is being removed from?

Reply:
Fixed. Thanks!